doc/faq.rst

```diff
@@ -61,6 +61,92 @@ reporting a bug, which should look something like this::
     skcuda:        Not found
 
 
+Why shouldn't I "pickle" my MNE-Python objects for later use?
```

Comment:
sounds like an odd way to frame the question ... how about: " I can't load my "pickled" object into MNE-Python after I upgraded." -- or something like that.